### PR TITLE
Release/1.0.6

### PR DESCRIPTION
**Pull request by HannesSommer**
I would strongly suggest to do a fast-forward here instead of a merge! 
I expect history ugliness and maybe even troubles (with some deployment tools) if the tag is pointing to second parent of a merge commit :). Or am I wrong?

And don't forget to copy or recreate the new tag 1.0.6.

### CODE REVIEW

**Comment by simonlynen** [comment]
ok to test

File: CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,19 @@
 Changelog for package libnabo
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+1.0.6 (2015-03-05)
+------------------
+* Reset point indices of results with distances exceeding threshold (#23, #24)
+* Fine tune the find_package() capability and add uninstall target (#22)
+* Fixed compiler warning (#18)
+* Added OpenMP support (#20, #21)
+* Build type tuing (#19)
```
Comment (simonlynen) [inline CHANGELOG.rst]: tuning

**Comment by simonlynen** [comment]
one comment, then this lgtm.

File: CHANGELOG.rst (hunk 1 above)
Comment (HannesSommer) [inline CHANGELOG.rst]: Oops, thanks.

**Comment by HannesSommer** [comment]
Ok, fixed the typo.

**Comment by simonlynen** [comment]
@HannesSommer thanks

**Comment by HannesSommer** [comment]
Wow, you really did a non ff merge on that? Thats exciting. Hopefully it works this way as well.

**Comment by HannesSommer** [comment]
Ah, I think you did not copy the tags, right?

**Comment by HannesSommer** [comment]
Could you quickly push the copy the tags or give me write access?

**Pull request merged**